app/src/main/java/com/example/myfridgehome/service/RecipeService.kt

```diff
@@ -9,8 +9,10 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class RecipeService {
+
+    //Creates recipes for application
```

Comment:
Added comments

Reply:
Change this to proper JavaDoc, above the class.  Add more value to the description... one can likely infer this comment from the name of the class.